### DISCUSSION TITLE
oh-my-posh.exe not running after installed

### DISCUSSION THREAD

**Discussion by morphiezy** [JanDeDobbeleer/oh-my-posh#2422] [Troubleshoot] [answered]
After successfully and restart windows terminal , i try to run my OMP with command "oh-my-posh.exe" but this is what happened .

![OMP](https://user-images.githubusercontent.com/71913929/174580926-d08264c7-928f-4c9e-8805-ae0b0b9fd53b.png)
![OMP_1](https://user-images.githubusercontent.com/71913929/174580940-f6effb85-86fd-4407-98e2-05918035bb8c.png)

**Comment by lewis-yeung** [comment, marked as answer]
@morphiezy See [docs](https://ohmyposh.dev/docs/installation/prompt).
